sdk/src/test/java/com/hedera/hashgraph/sdk/MnemonicTest.java

```diff
@@ -19,6 +19,8 @@
  */
 package com.hedera.hashgraph.sdk;
 
+import com.hedera.hashgraph.sdk.utils.Bip32Utils;
+import org.bouncycastle.util.encoders.Hex;
```

Comment:
Since this is the first implementation of the new standard, I think it would make sense to include a new set of standard "hedera" test vectors. The list I had come up with is detailed in the `Tests` section of [the standards issue](https://github.com/hashgraph/hedera-sdk-reference/issues/73#issue-1485459503), bullet #2

In addition to defining new test vectors, this would also involve renaming test variables `MNEMONIC3_STRING`, `MNEMONIC_LEGACY_STRING`, `MNEMONIC_STRING`, and `MNEMONIC_PRIVATE_KEY`, so that it is clear what these actually are.

Reply:
We've already added the test vectors (`toStandardED25519PrivateKey` and `toStandardECDSAsecp256k1PrivateKey` methods in `MnemonicTest`

Reply:
It's important that these particular test vectors are tracked somewhere so that all SDKs use the same ones. That will ensure that the `Standard` mnemonic->key functions are consistent across SDKs